packages/augur-ui/src/modules/trading/betslip.tsx

```diff
@@ -78,6 +80,14 @@ export const Betslip = () => {
           )}
         </section>
       </section>
+      <section className={Styles.SideImages}>
+        <a href="" target="_blank" rel="noopener noreferrer">
+          <img src={BannerSportsbook} alt="The image shows an American Football player running. The banner says '+1000 markets' and there is a button entitled 'Explore Sportsbook'."/>
+        </a>
+        <a href="" target="_blank" rel="noopener noreferrer">
+          <img src={BannerTrading} alt="The image shows Augur's Trading user interface and it says 'Trade Now!'."/>
+        </a>
+      </section>
```

Comment:
I don't think this will be the spot we put these. aka they don't belong to the `betslip` just because they are under it. However for now, this is fine, but it will definitely move.